server/src/main/resources/application.yml

```diff
@@ -38,3 +38,30 @@ datasources:
     password: 'postgres'
     schema-generate: CREATE_DROP
     dialect: POSTGRES
+
+---
+micronaut:
+  security:
+    enabled: true
+    authentication: idtoken
+    oauth2:
+      clients:
+        google:
+          client-id: ''
+          client-secret: ''
```

Comment:
Since these are defined now PR #78 lets go ahead and put the replacement tokens in here before merging this.